src/shared/actions/flags.ts

```diff
@@ -1,14 +1,17 @@
 import {Dispatch} from 'redux'
 import {getFlags as getFlagsRequest} from 'src/client'
 import {FlagMap} from 'src/shared/reducers/flags'
```

Comment:
i think this should be moved to a common file and included from there. as it is now, this creates a circular dependency between actions/flags and reducers/flags since they both import from each other.